java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java

```diff
@@ -172,6 +172,7 @@ public void startRayProcesses(boolean isHead) {
         startRedisServer();
       }
       startObjectStore();
+      startGcsServer();
```

Comment:
This should be put in the above `if isHead` section. Also, can you consolidate this with `startRedisServer` into a method called `startGcs`?

Reply:
fixed